### PR TITLE
added a built with page in the docs

### PR DESCRIPTION
**Pull request by tarekziade**
## Checklist

- [ ] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] If you provide code modification, please add yourself to `CONTRIBUTORS.txt`
  * The format is &lt;Name&gt; &lt;Surname&gt;.
  * Please keep alphabetical order, the file is sorted by names. 
- [ ] Add a new news fragment into the `changes` folder
  * name it `<issue_id>.<type>` for example (588.bug)
  * if you don't have an `issue_id` change it to the pr id after creating the pr
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files."

### CODE REVIEW

**Comment by asvetlov** [comment]
Thanks

File: docs/built_with.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Built with aiohttp
+==================
+
+aiohttp is used to build useful libraries built on top of it,
+and there's a page dedicated to list them: :ref:`aiohttp-3rd-party`.
```
Comment (socketpair) [inline docs/built_with.rst:5]: image

It seems this is wrong

Comment (asvetlov) [reply docs/built_with.rst:5]: Please feel free to propose better sentence.

Comment (socketpair) [reply docs/built_with.rst:5]: I mean wrong link. Wrong visual render and not clickable. Am I wrong ?

Comment (asvetlov) [reply docs/built_with.rst:5]: @socketpair would you fix it?

Comment (socketpair) [reply docs/built_with.rst:5]: Unfortunatelly I'm not an expert in RST. @jerryramires will try to fix.

Comment (tarekziade) [reply docs/built_with.rst:5]: @socketpair @asvetlov it's rendering well on http://aiohttp.readthedocs.io/en/latest/built_with.html

Notice that the "ref" markup is only rendered by Sphinx, not github's restructuredtext renderer

Comment (asvetlov) [reply docs/built_with.rst:5]: @tarekziade thanks for correction.
Now I finally understand why @socketpair raised the issue.
Nothing to do here.

**Pull request merged**